packages/ui/src/components/molecules/Dialog/DialogBox.tsx

```diff
@@ -66,6 +72,8 @@ const DialogBoxStyle = styled.section<DialogBoxProps>`
 `;
 
 const DialogBoxHeaderBarStyle = styled.div<DialogBoxUtilityProps>`
+  position: sticky;
+  top: 0;
```

Comment:
Do not modify base css for existing components, it'll break the current UI

Reply:
done